packages/quantic/jest.config.js

```diff
@@ -31,6 +31,11 @@ module.exports = {
       '<rootDir>/force-app/main/default/lwc/searchBoxStyle/searchBoxStyle',
     '^c/quanticFacetStyles$':
       '<rootDir>/force-app/main/default/lwc/quanticFacetStyles/quanticFacetStyles',
+    '^c/(.*)$': [
```

Comment:
What does this do?

Reply:
Trying to fix jest import paths, essentially telling jest that all paths that starts with `c/*` are _known_ I think?